konnectivity-client/pkg/client/client_test.go

```diff
@@ -540,3 +595,13 @@ func (s *proxyServer) handleData(pkt *client.Packet) *client.Packet {
 		},
 	}
 }
+
+// Override goleakVerifyNone to set t.Helper.
+// TODO: delete this once goleak has been updated to include
+// https://github.com/uber-go/goleak/commit/2dfebe88ddf19de216c4ab15a1189fc640b1ea9f
+func goleakVerifyNone(t *testing.T, options ...goleak.Option) {
```

Comment:
Consider making this part of a util in konn-client.

Reply:
Ack. Will follow up in a separate PR.